docs/tasks/semester-checklist.rst

```diff
@@ -1,9 +1,39 @@
 Semester Checklist
 ==================
-
 A quick checklist of tasks that need to be completed at the start of every semester.
 
--  Reserve the meeting room by speaking to the lab manager (requires a signature from the president and vice president). Get set up with swipe access if need be.
+Before Semester
+---------------
+Tasks to do during the break.
+
+-  Hold weekly/bi-weekly eboard meetings.
+-  Discuss plans and brainstorm for next semester.
+-  Reserve meeting space.
+-  Triage and review tasks from previous semester to plan for coming semester (see project board).
+
+Before Fall Semester
+----------------------
+Tasks which only need to be completed before the fall semester.
+
+-  Get all eboard members financially certified through Campus Life.
+-  Plan for the Activities Gala.
```

Comment:
Let's cross-link to our Runbook page on club fairs / Activities Gala here. :smile: To do a cross-document link in Sphinx, type the it like this:

```rest
Plan for the :doc:`Activities Gala <../events/club-fairs>`.
```